apps/judicial-system/web/src/routes/Court/components/CourtCaseNumber/CourtCaseNumber.tsx

```diff
@@ -61,7 +63,7 @@ const CourtCaseNumber: React.FC<Props> = (props) => {
 
     await updateCase(id, update)
     if (update.courtCaseNumber) {
-      receiveCase(workingCase, update.courtCaseNumber)
+      sendNotification(workingCase.id, NotificationType.RECEIVED_BY_COURT)
```

Comment:
This will also send a notification when the court case number is changed which we do not want to do.

Since the transition of the case has moved to the server, it would make sense to send the notification from there as well.

Consider moving the notification to the server. Alternatively, we could send the notification only when the state was SUBMITTED before the court case number update.